packages/evolution-common/src/services/baseObjects/BaseTrip.ts

```diff
@@ -70,20 +70,62 @@ export class BaseTrip extends Uuidable implements IValidatable {
      * Factory that validates input from an interview and makes
      * sure types and required fields are valid before returning a new object
      * @param dirtyParams
-     * @returns BaseTrip
+     * @returns BaseTrip | Error[]
```

Comment:
Is `Error[]` enough? What will you do with it in the end? Just dump it somewhere for the analyst to see? Or would you need some more specific error type, with object class, object uuid and error message?